listen/property-changes.js

```diff
@@ -312,7 +341,8 @@ PropertyChanges.prototype.makePropertyObservable = function (key) {
                 }
                 PropertyChanges.dispatchBeforeOwnPropertyChange(this, key, overriddenDescriptor.value);
                 overriddenDescriptor.value = value;
-                state[key] = value;
+				//Benoit: What's the point of this?
+                //state[key] = value;
```

Comment:
This note prevents a memory leak by only permitting a property observer thunk to be installed once.